core/src/main/java/org/stellar/anchor/paymentservice/PaymentHistory.java

```diff
@@ -1,11 +1,17 @@
 package org.stellar.anchor.paymentservice;
 
+import java.util.ArrayList;
 import java.util.List;
+import lombok.Data;
 
-@SuppressWarnings("unused")
+@Data
 public class PaymentHistory {
```

Comment:
There is some overlap between your changes in this PR and [mine](https://github.com/stellar/java-stellar-anchor-sdk/pull/28/files) for the StellarPaymentService.